### PR TITLE
Upgrade to PHP 5.6, required as of Shopware 5.2

### PR DESCRIPTION
**Pull request by sebastien-fauvel**
Addresses https://github.com/shopwareLabs/shopware-vagrant/issues/5

### CODE REVIEW

**Comment by sebastien-fauvel** [comment]
@bcremer please review

File: README.md [hunk 1]
```diff
@@ -34,18 +34,64 @@ To SSH into the created VM:
...
+Configure your online store in a web browser with the credentials demo/demo:
+
+- Backend: [http://33.33.33.10/shopware/backend/](http://33.33.33.10/shopware/backend/)
+
+You can then access your storefront at:
+
+- Front-end: [http://33.33.33.10/shopware/](http://33.33.33.10/shopware/)
 
-## Installation under arch linux
+## Installation under Linux
```
Comment (dnoegel) [inline README.md]: the pacman part is arch specific, so the "arch linux" headline was intentional

**Comment by sebastien-fauvel** [comment]
@dnoegel sorry about that – please review and merge at your convenience

**Comment by dnoegel** [comment]
@sebastien-fauvel Thanks for your work!

**Comment by sebastien-fauvel** [comment]
You're welcome :)

**Comment by bcremer** [comment]
@sebastien-fauvel Thank you :+1:

**Pull request merged**